### DISCUSSION TITLE
支持不同应用使用不同路由规则

### DISCUSSION THREAD

**Discussion by Davidasx** [MetaCubeX/ClashMetaForAndroid#397] [Ideas]
例如为了保护隐私，对一些应用使用全部reject模式（阻止联网上传数据），对一些应用使用正常模式
配置文件可以改，但是不方便，求一个应用内配置不同应用不同路由的办法

**Comment by Davidasx** [comment]
这是防止监听的很好的办法

**Comment by huangdog** [comment]
kr328的clash for android （原版）支持PROCESS-NAME 规则，根据app给相应的路由规则。反而非原版的客户端，目前来说统统没有。

**Reply by Davidasx** [reply]
可惜删库了
